rts/lra/client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java

```diff
@@ -449,11 +449,13 @@ public void leaveLRA(URI lraId, String body) throws WebApplicationException {
     }
 
     /**
-     * For particular compensator class it returns termination uris based on the provided base uri.
-     * You get map of string and URI.
+     * For particular compensator class it returns termination uris based on the
+     * provided base uri. You get map of string and URI.
```

Comment:
Please can I ask why the new line was introduced before the `provided base uri`?

Reply:
That is a mistake, I will fix that and I will add an extra change so that I can address the problem related to javadoc unknown tag "lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java:728: error: unknown tag: LraId
[ERROR]      * PUT on {@value LRAConstants#COORDINATOR_PATH_NAME}/<LraId>/remove with the URL of the participant."
The javadoc version is 21